### PR TITLE
Tbalo/feat/issue 14 and 15

### PR DESCRIPTION
**Pull request by TBalo**
### Questions
42. What are Concurrency and Parallelism?
43. What is a shallow copy?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ In writing the answers I was helped by AI and the book "JavaScript: The Definiti
 |39. [What is a module?](#what-is-a-module?)|
 |40. [What is a Web API?](#what-is-a-web-api?)|
 |41. [What is Hydration?](#what-is-hydration?)|
+|42. [What are Concurrency and Parallelism?](#What-are-Concurrency-and-Parallelism?)|
```
Comment (ir3ne) [inline README.md:60]: please write the anchor tags in lowercase as the others

File: README.md [hunk 2]
```diff
@@ -2774,3 +2776,104 @@ In short, HTML rendering is indeed the first step, followed by the JavaScript do
 </div>
 
 ---
+
+<a name="What-are-Concurrency-and-Parallelism?"></a>
+
+# Concurrency and Parallelism in JavaScript
```
Comment (ir3ne) [inline README.md:2782]: please add also the number of the question before the title as the others

File: README.md [hunk 3]
```diff
@@ -2774,3 +2776,104 @@ In short, HTML rendering is indeed the first step, followed by the JavaScript do
 </div>
 
 ---
+
+<a name="What-are-Concurrency-and-Parallelism?"></a>
```
Comment (ir3ne) [inline README.md:2780]: then lowercase also here

File: README.md [hunk 4]
```diff
@@ -2774,3 +2776,104 @@ In short, HTML rendering is indeed the first step, followed by the JavaScript do
...
+
+[back to Questions](#questions)
+
+</div>
+
+---
+
+<a name="What-is-a-shallow-copy?"></a>
+
+# What is a Shallow Copy?
```
Comment (ir3ne) [inline README.md:2847]: add the question number before the title

File: README.md (hunk 2 above)
Comment (ir3ne) [inline README.md:2782]: all the titles are heading 2 not one so add `##` before the title

File: README.md (hunk 4 above)
Comment (ir3ne) [inline README.md:2847]: all the titles are heading 2 not one so add `##` before the title

File: README.md [hunk 5]
```diff
@@ -2774,3 +2776,104 @@ In short, HTML rendering is indeed the first step, followed by the JavaScript do
...
+
+A **shallow copy** is a duplicate of an object where the top-level properties are copied, but the nested objects or arrays are shared between the original and the copy. This means that if you modify a nested object in the copy, the change will also affect the original object, since both reference the same nested object.
+
+## Shallow Copy in JavaScript
```
Comment (ir3ne) [inline README.md:2851]: you can remove this extra title since we know we're talking about js

File: README.md [hunk 6]
```diff
@@ -2774,3 +2776,104 @@ In short, HTML rendering is indeed the first step, followed by the JavaScript do
...
+
+In JavaScript, you can create a shallow copy of an object using several methods. Here are a few common techniques:
```
Comment (ir3ne) [inline README.md:2853]: all good but I would add a paragraph explaining when shallow copy is useful

File: README.md [hunk 7]
```diff
@@ -2774,3 +2776,104 @@ In short, HTML rendering is indeed the first step, followed by the JavaScript do
 </div>
 
 ---
+
+<a name="What-are-Concurrency-and-Parallelism?"></a>
+
+# Concurrency and Parallelism in JavaScript
+
+In JavaScript, concurrency and parallelism both enable handling multiple tasks but have different approaches.
```
Comment (ir3ne) [inline README.md:2784]: all good here. I would add a line explaining when to prefer using concurrency and when parallelism

**Comment by TBalo** [comment]
@ir3ne.  I've made the changes..

**Review by ir3ne — APPROVED** [review]

**Pull request merged**